### ISSUE TITLE
about test and train consistency

### ISSUE THREAD

**Issue by zhaoedf** [Continvvm/continuum#165]
in docs [https://continuum.readthedocs.io/en/latest/_tutorials/scenarios/scenarios.html](url), i see how i can both train an test my model.
we all know that in incremental learning context, when doing test, you can only use the classes that have already been seen.
however, it seems there is no  guarantee in that code that the order of train tasks is the same as test tasks. Am i right?

**Comment by TLESORT** [comment]
Hi @zhaoedf , thanks for your question :)

The incremental scenario class is deterministic. Two scenarios created with the same parameters will have the same order of classes in tasks, therefore if you create one scenario with train data and another with test data, both will have the same classes in the same tasks.

I hope it answers your question, do not hesitate to reopen the issue if it does not.

(Btw there is no problem if the test set contains data about the future as long as the test set is not used for hyper-parameters / model selection. )
